var/spack/repos/builtin/packages/r-splitstackshape/package.py

```diff
@@ -0,0 +1,46 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class RSplitstackshape(RPackage):
+    """Stack and Reshape Datasets After Splitting Concatenated Values.
+
+    Online data collection tools like Google Forms often export
+    multiple-response questions with data concatenated in cells. The
+    concat.split (cSplit) family of functions splits such data into
+    separate cells. The package also includes functions to stack groups
+    of columns and to reshape wide data, even when the data are
+    "unbalanced" something which reshape (from base R) does not handle,
+    and which melt and dcast from reshape2 do not easily handle.
+    """
+
+    homepage = "http://github.com/mrdwab/splitstackshape"
+    url      = "https://cran.r-project.org/src/contrib/splitstackshape_1.4.4.tar.gz"
+
```

Comment:
Can you add a `list_url`? Otherwise, as soon as a new version comes out, this package will no longer be able to download 1.4.4.
```python
list_url = "https://cran.r-project.org/src/contrib/Archive/splitstackshape/"
```